ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -142,6 +142,8 @@ QtObject {
 
     property bool isCommunityHistoryArchiveSupportEnabled: advancedModule? advancedModule.isCommunityHistoryArchiveSupportEnabled : false
 
+    property bool isWakuV2StoreEnabled: advancedModule ? advancedModule.isWakuV2StoreEnabled: false
```

Comment:
Second `:` is looking like is a property assignment and not a part of `? :` statement. I suggest adding a space :)